shell.nim

```diff
@@ -220,9 +259,26 @@ proc asgnShell*(cmd: string): tuple[output: string, exitCode: int] =
         # outstream died on us?
         doAssert outStream.isNil
         break
+
+    if not outStream.atEnd():
+      if dokOutput in debugConfig:
+        let rem = outStream.readAll()
+        res &= rem
+        for line in rem.split("\n"):
+          echo "shell> ", line
+      else:
+        res &= outStream.readAll()
+
     let exitCode = pid.peekExitCode
     if exitCode != 0:
       # add error stream to output
+
+      # NOTE TODO instead of adding error and result into the same
+      # output (making it impossible to differentiate between them) it
+      # might be possible to either change function signature (not
+      # desired as it breaks API) or provide optional parameter for
+      # function call (`var errorRes: Option[string]` or something
+      # like that)
```

Comment:
I wouldn't worry too much about breaking the API tbh. `asgnShell` wasn't intended as a user facing proc in the first place. It's mainly exported for the tests, iirc. 

And if anyone used `asgnShell` manually, that's what versioning is for (I'd release version v0.3.0 with your PR for that reason). So feel free to return 
```nim
tuple[output, error: string, exitCode: int]
```
instead.